src/tests/suite/daos_container.c

```diff
@@ -1881,6 +1881,157 @@ co_owner_implicit_access(void **state)
 	test_teardown((void **)&arg);
 }
 
+static void
+expect_co_set_attr_access(test_arg_t *arg, uint64_t perms, int exp_result)
+{
+	daos_prop_t	*cont_prop;
+	int		 rc = 0;
+	const char	*name = "AttrName";
+	const char	*value = "This is the value";
+	const size_t	 size = strnlen(value, 32);
```

Comment:
maybe consider a constant that both this function to set an attr and the expect_co_get_attr_access() function use to constrain the size of attribute values consistently?

Reply:
That's fair. I went back and forth with myself about the idea since there wasn't a publicly exposed constant (that I could find, anyway), and static analysis dings us for using plain strlen, even though we know the string is null terminated in this case. Will change if there are any issues that require a re-push.

Reply:
could also change to something like this to avoid strnlen altogether

const char value[] = "This is the value";
const size_t size = sizeof(value) - 1;

Reply:
Good suggestion, Jeff. I'll fix both of Ken's comments.

Reply:
For some reason I end up with a segfault down in mercury when I switch value to being a static char array, with no other changes. Weird because they should be functionally equivalent. I'm just going to use a const max with strnlen.